src/js/ImportController.js

```diff
@@ -53,15 +50,13 @@ goog.require('ga_wmts_service');
         'http://wms.geo.gr.ch/wms/amtlichevermessung_stand',
         'http://wms.geo.gr.ch/wms/wildruhezonen',
         'http://wms.geo.gr.ch/wms/wildschutzgebiete',
-        'http://wms.geo.gr.ch/wms/zonenplan',
         'http://www.sogis1.so.ch/wms/avwms',
         'http://www.sogis1.so.ch/wms/grundbuchplan',
         'http://www.sogis1.so.ch/wms/wms_lidar',
         'http://wms.vd.ch/public/services/wmsVD/Mapserver/Wmsserver',
-        'https://wms.geo.bs.ch/wmsBS',
+        'http://wms.geo.bs.ch/wmsBS?',
```

Comment:
? is not necessary

Reply:
it _IS_ necessary ..othrwise it wont load in the pulldown....see the curretn implementation

Reply:
I've just tested on prod it works without. I think the WMS is just very  unstable.

Reply:
according to OWSLib this one works always with ? ....